osrm 5.8.0 doesn't build for node v7

Installing osrm 5.8.0 fails on node v7 with the following error:
```
-- Building node_osrm
CMake Error at src/nodejs/CMakeLists.txt:11 (include):
  include could not find load file:

    NodeJS

CMake Error at src/nodejs/CMakeLists.txt:12 (nodejs_init):
  Unknown CMake command "nodejs_init".
```
Using node 7.10:
```
$ node -v
v7.10.0
```
and installing osrm 5.8.0 as such:
```
 "dependencies": {
    "osrm": "^5.8.0"
    }
```

However using node v6 there is no issue
```
$ node -v
v6.11.0
```

```
$ npm install osrm

> osrm@5.8.0 install /path/node_modules/osrm
> node-pre-gyp install --fallback-to-build=false || ./scripts/node_install.sh

[osrm] Success: "/path/node_modules/osrm/lib/binding/node_osrm.node" is installed via remote
```

--- a/package.json
+++ b/package.json
@@ -53,6 +53,8 @@
     "xmlbuilder": "^4.2.1"
   },
   "bundleDependencies": [
+    "nan",
+    "node-cmake",
     "node-pre-gyp"
   ],
   "main": "lib/index.js",

